### DISCUSSION TITLE
No client-side Javascript allowed!

### DISCUSSION THREAD

**Discussion by Janaka-Steph** [RyanClementsHax/tailwindcss-themer#125] [General]
I would like to have a theme toggle that works even when no client-side Javascript is allowed!
I saw that I can specify a selector. I was thinking it could be based on a hidden checkbox input which has two states: checked / not checked. If checked => darkmode, not checked => light mode.
Do you think it is doable?

**Comment by RyanClementsHax** [comment]
Thanks for your thoughts!

I think some people would find this helpful.

Yes! I do think this is doable. It's just a matter of crafting a specific selector for the checked checkbox :)

**Reply by Janaka-Steph** [reply]
Nice! I will try, thank you!
